src/colorizer/Collection.ts

```diff
@@ -200,6 +200,17 @@ export default class Collection {
     return this.formatDatasetPath(collectionDirectory + "/" + datasetPath);
   }
 
+  // TODO: Refactor how dummy collections store URLs? The URL should always be a valid resource maybe?
```

Comment:
I'm not understanding the TODO. What is a dummy collection?  Is that when user tried to load a dataset directly and you have to internally build a fake collection to hold it?

Reply:
Correct! Currently, when I create a dummy collection I set the collection's base URL to `null`, and the only URL data that is provided is in the single dataset it wraps. I think I did this so I wouldn't be duplicating the URL, but the only reason that `null` value is stored is so I can tell if it's a dummy collection. It would likely be much clearer to separate that behavior out into its own flag.